### PR TITLE
Adds links to TechFAR and Digital Services Playbook

### PR DESCRIPTION
**Pull request by OpenGlobe**
Adds links to TechFAR and Digital Services Playbook

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ layout: homepage
   <div class="usa-grid">
     <h2>Introducing the Contracting Cookbook</h2>
     <p>The goal of the digital contracting cookbook is to provide agencies with information and suggestions about how to acquire digital services, based on the authors' experience. The cookbook is not a "how-to," in the sense that agencies' requirements are different and there are multiple ways to achieve success.</p>
+    <p>The <a href="https://playbook.cio.gov/techfar/" target="_blank">TechFAR Handbook</a> is yet another handy resource.  Part of the <a href="https://playbook.cio.gov/" target="_blank">Digital Services Playbook</a>, the TechFAR Handbook expands on software development strategies and procurement insights.  Consider both of these resources as additional helpful guides for your use.
```
Comment (khandelwal) [inline index.md:37]: Might be useful to spell out what the TechFAR is (what does that word-acronym combo mean) and when one might want to use the handbook (provide context).

**Comment by khandelwal** [comment]
@vzvenyach merged this in - but I think I still want my comment answered (or tell me why it's not necessary)

**Comment by vdavez** [comment]
@khandelwal I will add. That's why i didn't close.

**Comment by vdavez** [comment]
Oh crap. I _did_ close. Will add.

**Pull request merged**